README.md

```diff
@@ -27,18 +27,36 @@ import 'package:mp_audio_stream/mp_audio_stream.dart';
 void main() async {
   final audioStream = getAudioStream();
 
-  audioStream.init();
+  //default init params: {int bufferMilliSec = 3000,
+  //                      int waitingBufferMilliSec = 100,
+  //                      int channels = 1,
+  //                      int sampleRate = 44100}
+  audioStream.init( {channels: 2} ); //Call this from Flutter's State.initState() method
```

Comment:
@hollingsworthd, thank you so much for your great pull request!

Would you mind adjusting the syntax here to use `init(channels: 2)` instead of `init({channels:2})`? I believe the latter isn't consistent with Dart's syntax. I've been out of the Flutter loop for a bit, so I might not be up to date with the latest changes. Thanks!